docker/package/wizard_structure.py

```diff
@@ -14,6 +14,14 @@
 import urllib.request, urllib.parse
 import json
 
+# Currently available networks
+
+networks = {
+    "mainnet": "Main Tezos network",
+    "hangzhounet": "Test network using version 011 of Tezos protocol (Hangzhou)",
+    "ithacanet": "Test network using version 012 of Tezos protocol (Ithaca2)",
+}
+
```

Comment:
One last thing: there's a section `# Global options` further down, please move this there for ease of reference.